lib/3scale/api/client.rb

```diff
@@ -623,6 +623,107 @@ def delete_service_feature(service_id, id)
         true
       end
 
+      # @api public
+      # @param [Fixnum] account_id Account ID
+      # @param [String] state State
+      # @param [String] role Role
+      # @return [Array<Hash>]
+      def list_users(account_id, state: nil, role: nil)
+        params = { state: state, role: role }.reject { |_, value| value.nil? }
```

Comment:
Thank you for the feedback, I decided not to change it simply because the find_application method also uses this construct. So opted to keep it consistent.

Reply:
unit test for activate_user is added